admin-support-cli/tests/demotion-report-action.test.js

```diff
@@ -253,15 +254,16 @@ describe('Demotion report', () => {
         issueNumber: 1,
         duration: 1,
         ticket: '1234',
-        demotionDate: new Date(Date.parse('2021-02-08T10:20:00Z')),
-        promotionDate: new Date(Date.parse('2021-02-07T10:20:00Z')),
+        demotionDate: '2021-02-08T10:20:00Z',
+        promotionDate: '2021-02-07T10:20:00Z',
```

Comment:
@stebje I changed these dates back to the ISO-8601 form since the previous version was creating a more human-readable date string that seemed to break the test. Could this be related to the shift from `nock` to `fetch-mock`? Not sure if this fix is correct but it seems to get things working again.